src/Sound/Tidal/UI.hs

```diff
@@ -43,10 +57,19 @@ import           Sound.Tidal.Utils
 ------------------------------------------------------------------------
 -- * UI
 
--- | Randomisation
+-- ** Randomization
```

Comment:
I'm pretty self-conscious about US spellings in Tidal docs, but I don't know if it's something the Brits/Europeans in the community actually care about 🤷